packages/cli/bin/comet.js

```diff
@@ -1,3 +1,6 @@
 #!/usr/bin/env node
-require("ts-node").register();
+require("ts-node").register({
+    transpileOnly: true, // increases ramp up time of generator - see: https://github.com/TypeStrong/ts-node?tab=readme-ov-file#transpileonly
```

Comment:
This works not as expected, see type field in product.entity. this is an enum field and will be ignored now for generation maybe because of the transpiling only flag. I think this flag cannot be activated here.

Reply:
Indeed, looks like this has problems with enums 😢